src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -61,6 +61,7 @@ private static bool IsStringToken(SyntaxToken token)
                 || token.IsKind(SyntaxKind.CharacterLiteralToken)
                 || token.IsKind(SyntaxKind.InterpolatedStringStartToken)
                 || token.IsKind(SyntaxKind.InterpolatedVerbatimStringStartToken)
+                || token.IsKind(SyntaxKind.VerbatimInterpolatedStringStartToken)
```

Comment:
this is hte part i was worried about.  basically, we have to check every usage of InterpolatedVerbatimStringStartToken to add the alternate.  I don't think tehre's a lot of value in an actual syntax kind her.e  I think it should be one kind, doc'ed as "the start token will either start with @$ or $@"